web-apps/genesys-spark-examples/Jenkinsfile

```diff
@@ -4,9 +4,9 @@ Boolean isMainBranch = env.BRANCH_NAME == 'main'
 
 Boolean isMaintenanceReleaseBranch = env.BRANCH_NAME.startsWith('maintenance/')
 
-Boolean isFeatureBranch = env.BRANCH_NAME.startsWith('feature/') || env.BRANCH_NAME.startsWith('beta/')
+Boolean isFeatureBranch = env.BRANCH_NAME.startsWith('feature/') || env.BRANCH_NAME.startsWith('beta/') || isMainBranch
```

Comment:
So, if I am understanding this correctly. This is to maintain the current behavior after migrating the `beta/v4` branch to main?